aiida/backends/tests/cmdline/commands/test_code.py

```diff
@@ -96,6 +96,27 @@ def test_noninteractive_upload(self):
         self.assertClickResultNoException(result)
         self.assertIsInstance(Code.get_from_string('{}'.format(label)), Code)
 
+    def test_from_config(self):
+        """Test setting up a code from a config file"""
+        from aiida.orm import Code
+        import tempfile
+        import os
+
+        label = 'noninteractive_config'
+
+        with tempfile.NamedTemporaryFile('w', delete=False) as handle:
```

Comment:
I think you are not deleting the test config files here, correct?

Reply:
thanks for pointing this out, fixed (also in other places)